client/src/app/core/repositories/management/meeting-repository.service.ts

```diff
@@ -85,7 +86,7 @@ export class MeetingRepositoryService extends BaseRepository<ViewMeeting, Meetin
             dashboard: dashboardFields,
             settings: this.meetingSettingsDefinitionProvider
                 .getSettingsKeys()
-                .concat(`jitsi_domain`, `jitsi_room_name`, `jitsi_room_password`),
+                .concat(`jitsi_domain`, `jitsi_room_name`, `jitsi_room_password`, `enable_chat`),
```

Comment:
I don' really get why this is an option. Chat could just be enabled all the time, no?

Reply:
This should be discussed with @emanuelschuetze.